examples/syncing/syncing.cc

```diff
@@ -1,40 +1,40 @@
-#include <cstdio>
 #include <vector>
 
 #include <celerity.h>
```

Comment:
Well it used to be the case that we couldn't do `<celerity/celerity.h>` because that folder structure only exists once Celerity is installed. However now that you mention it I'm somewhat confused as to how this even works with the changes you made in #51..?!

Reply:
It works because we include this directory explicitly in the top-level CMakeLists:
```cmake
target_include_directories(celerity_runtime PUBLIC
  ...
  $<INSTALL_INTERFACE:include/celerity>
)
```

But you're right, changing that would break the non-installed build. Let's do that at another point in time.